quilt/tools/build.py

```diff
@@ -105,3 +118,95 @@ def build_package(username, package, yaml_path):
         _build_table(build_dir, store, '', tables)
         if readme is not None:
             _build_file(build_dir, store, 'README', rel_path=readme)
+
+def splitext_no_dot(filename):
+    name, ext = os.path.splitext(filename)
+    ext.strip('.')
+    return name, ext.strip('.')
+
+def generate_build_file(startpath, outfilename='build.yml'):
+    startbase = os.path.basename(startpath)
+    buildfiles = {startbase : {}}
+    buildtables = {startbase : {}}
+
+    def add_to_buildfiles(path, files):
+        try:
+            safepath = [_pythonize_name(d) if d != '.' else '.' for d in path]
```

Comment:
So how do you end up with a '.'? `path` comes from `os.path.relpath`, right? I can't get it to return anything that starts with "./".